### PR TITLE
{lang}[GCCcore/10.2.0] Perl v5.32.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node3405.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/9a74fd38346a7fb082e4f3f0a68ad94c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2609.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/c6674738c4420cb0fcdbe345f47b70bf for a full test report.

File: easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,1747 @@
...
+        'checksums': ['51220fcaf9f66a639b69d251d7b0757bf4202f4f9debd45bdd341a6aca62fe4e'],
+    }),
+    ('Encode::Locale', '1.05', {
+        'source_tmpl': 'Encode-Locale-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/G/GA/GAAS'],
+        'checksums': ['176fa02771f542a4efb1dbc2a4c928e8f4391bf4078473bd6040d8f11adb0ec1'],
+    }),
+    ('XML::SAX::Base', '1.09', {
+        'source_tmpl': 'XML-SAX-Base-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/ / "/ "GRANTM"'],
```
Comment (smoors) [inline easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb:251]:
```suggestion
        'source_urls': ['https://cpan.metacpan.org/authors/id/G/GR/GRANTM'],
```

File: easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,1747 @@
...
+        'checksums': ['9063b3c346edeb422807ffe49ffb23038c4f900d4a77b845ce4b53d97bf29400'],
+    }),
+    ('Tie::Function', '0.02', {
+        'source_tmpl': 'Tie-Function-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DA/DAVIDNICO'],
+        'checksums': ['0b1617af218dfab911ba0fbd72210529a246efe140332da77fe3e03d11000117'],
+    }),
+    ('Template::Plugin::Number::Format', '1.06', {
+        'source_tmpl': 'Template-Plugin-Number-Format-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/ / "/ "DARREN"'],
```
Comment (smoors) [inline easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb:321]: this one is also wrong?

File: easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,1747 @@
...
+        'checksums': ['81e4485ff3faf9b7813584d57b557f4b34e73b6c2eb696394f6deefacf5ca65b'],
+    }),
+    ('Sub::Uplevel', '0.2800', {
+        'source_tmpl': 'Sub-Uplevel-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN'],
+        'checksums': ['b4f3f63b80f680a421332d8851ddbe5a8e72fcaa74d5d1d98f3c8cc4a3ece293'],
+    }),
+    ('XML::Bare', '0.53', {
+        'source_tmpl': 'XML-Bare-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/ / "/ "CODECHILD"'],
```
Comment (smoors) [inline easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb:586]: idem

File: easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb [hunk 4]
```diff
@@ -0,0 +1,1747 @@
...
+        'checksums': ['2a48fa779b5297e56156380e8b32637c6c58decb4f4a7f3c7350523e11275f8f'],
+    }),
+    ('LWP::Simple', '6.47', {
+        'source_tmpl': 'libwww-perl-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/O/OA/OALDERS'],
+        'checksums': ['3029d6efc2099c4175bf976d0db7fbab9771ada631010c809cb4664230898f53'],
+    }),
+    ('Time::Piece::MySQL', '0.06', {
+        'source_tmpl': 'Time-Piece-MySQL-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/ / "/ "KASEI"'],
```
Comment (smoors) [inline easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb:731]: idem

File: easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb [hunk 5]
```diff
@@ -0,0 +1,1747 @@
...
+        'checksums': ['faddf43b4bc21db8e4c0e89a26e5f23fe626cde3491ec651b6aa338627f5775a'],
+    }),
+    ('List::AllUtils', '0.16', {
+        'source_tmpl': 'List-AllUtils-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DR/DROLSKY'],
+        'checksums': ['559b3aa911c73003a3a1ebd860d3b16e171137de8203d86be63a2390364c63dd'],
+    }),
+    ('UNIVERSAL::moniker', '0.08', {
+        'source_tmpl': 'UNIVERSAL-moniker-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/ / "/ "KASEI"'],
```
Comment (smoors) [inline easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb:916]: idem

File: easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb [hunk 6]
```diff
@@ -0,0 +1,1747 @@
...
+        'checksums': ['0e3001207496dda5d92a98e43020bd55864513408740059f869eab3b401c8ed3'],
+    }),
+    ('Math::CDF', '0.1', {
+        'source_tmpl': 'Math-CDF-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/C/CA/CALLAHAN'],
+        'checksums': ['7896bf250835ce47dcc813cb8cf9dc576c5455de42e822dcd7d8d3fef2125565'],
+    }),
+    ('Array::Utils', '0.5', {
+        'source_tmpl': 'Array-Utils-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/Z/ZM/ZMIJ'],
```
Comment (smoors) [inline easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb:1192]: maybe?
```suggestion
        'source_urls': ['https://cpan.metacpan.org/authors/id/Z/ZM/ZMIJ/Array'],
```

File: easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb [hunk 7]
```diff
@@ -0,0 +1,1747 @@
...
+        'checksums': ['45ebd8e64a54a86f88d8e01ae55212967c8aa8fed57e814085def7608ac65804'],
+    }),
+    ('File::pushd', '1.016', {
+        'source_tmpl': 'File-pushd-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN'],
+        'checksums': ['d73a7f09442983b098260df3df7a832a5f660773a313ca273fa8b56665f97cdc'],
+    }),
+    ('MooseX::Types::Perl', '0.101343', {
+        'source_tmpl': 'MooseX-Types-Perl-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/ / "/ "RJBS"'],
```
Comment (smoors) [inline easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb:1302]: idem

File: easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb [hunk 8]
```diff
@@ -0,0 +1,1747 @@
...
+        'checksums': ['ef620c1e019daf9cf3f23a943d25a94c91e93ab312bcd63be2e9740ec0b94288'],
+    }),
+    ('Mixin::Linewise::Readers', '0.108', {
+        'source_tmpl': 'Mixin-Linewise-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/R/RJ/RJBS'],
+        'checksums': ['7df20678474c0973930a472b0c55e3f8e85b7790b68ab18ef618f9c453c8aef2'],
+    }),
+    ('Config::INI', '0.025', {
+        'source_tmpl': 'Config-INI-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/ / "/ "RJBS"'],
```
Comment (smoors) [inline easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb:1402]: idem

**Review by smoors — DISMISSED** [review]
@boegel lots of wrong urls here

**Comment by verdurin** [comment]
@boegel could you look at the suggested changes please?

**Comment by boegel** [comment]
@verdurin done in e32ab31de9

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--force-download --sourcepath /tmp/$USER/pr11201"

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 13 out of 13 (4 easyconfigs in this PR)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.5
See https://gist.github.com/aeeadc21f396453489ad0ce305720be0 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

File: easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb (hunk 6 above)
Comment (verdurin) [reply easybuild/easyconfigs/p/Perl/Perl-5.32.0-GCCcore-10.2.0.eb]: Now fixed.

**Comment by verdurin** [comment]
@smoors are you content with the changes now?

**Comment by boegel** [comment]
@verdurin The changes were necessary to make downloading work, that's 100% fixed now (see how `boegelbot` tested this PR).

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**